cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -45,6 +45,10 @@
   // for backwards compatibility
   private final List<ETLStage> actions;
   private final Boolean service;
+  // support for SQL Engines
+  private final Boolean transformationPushdown;
+  private final String transformationPushdownType;
```

Comment:
does this correspond to the plugin type? If so, it should always be SQLEngine.PLUGIN_TYPE rather than user specified. If not, what is the type here?

Reply:
This is a future provision for additional Pushdown Modes (as defined in the PRD). This will be used for source-side and sink-side pushdown in the future.

I will remove this from this first release and add it back when needed at a later phase.